response_operations_ui/views/passwords.py

```diff
@@ -0,0 +1,141 @@
+import logging
+
+from flask import abort, redirect, render_template, request, url_for, Blueprint, current_app as app
+from itsdangerous import URLSafeSerializer, BadSignature, SignatureExpired, BadData
+from structlog import wrap_logger
+
+from response_operations_ui.forms import ForgotPasswordForm, ResetPasswordForm
+from response_operations_ui.controllers import uaa_controller
+from response_operations_ui.controllers.notify_controller import NotifyController
+from response_operations_ui.exceptions.exceptions import NotifyError
+from response_operations_ui.common import token_decoder
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+BAD_AUTH_ERROR = 'Unauthorized user credentials'
+
+passwords_bp = Blueprint('passwords_bp', __name__, static_folder='static', template_folder='templates')
+
+
+@passwords_bp.route('/forgot-password', methods=['GET'])
```

Comment:
My hero for using single functions for GET and POST. In a few places we handle both in the same function >_<